hurumap_land/templates/homepage.html

```diff
@@ -15,40 +15,87 @@
 </li>
 {% endblock header_menu %}
 
-{% block header_content %}
-<div class="content-container clearfix bg-color-secondary text-center">
-  <div class="wrapper">
-    <h1>Discover the Data behind our Land</h1>
-    <h4>{{ HURUMAP.name }} provides useful facts and data about land ownership in South Africa.</h4>
-  </div>
-</div>
-{% endblock header_content %}
-
-{% block how_it_works %}
-<div class="wrapper clearfix how-it-works">
-  <h1>How It Works</h1>
-  <br/>
-  <div class="column-third">
-    <p><span class="img-bg rounded bg-color-primary"><img src="{% static 'img/icons/blurbs/checklist.svg' %}"/></span></p>
-    <h3>Find Facts</h3>
-    <p>Land ownership data is broken down by province and category: agricultural land, erven and urban land and sectional title land. </p>
-  </div>
-  <div class="column-third">
-    <p><span class="img-bg rounded bg-color-primary"><img src="{% static 'img/icons/blurbs/analysis.svg' %}"/></span></p>
-    <h3>Visualize</h3>
-    <p>Our library of charts gives you insight into land data from provinces you research. Look for them on profile pages. You can embed the charts on your own site. </p>
+{% block header_content %}{% endblock header_content %}
```

Comment:
Please add comment here on what's happening.

Reply:
@KhadijaMahanga Yip, I figured as much.

The in-line/in-code comment is for future devs to understand the same thing quickly too.